python_modules/libraries/dagster-airbyte/dagster_airbyte_tests/experimental/conftest.py

```diff
@@ -147,7 +149,7 @@
 SAMPLE_DESTINATION_DETAILS = {
     "destinationId": TEST_DESTINATION_ID,
     "name": "My Destination",
-    "sourceType": "postgres",
+    "destinationType": TEST_DESTINATION_TYPE,
```

Comment:
There was a mismatch in the json example and the documentation for [get destination details](https://reference.airbyte.com/reference/getdestination) in the Airbyte API - the json example uses the key `sourceType`, but the correct key is `destinationType`

Reply:
Thanks for clarifying.